python/fazan/fazan.py

```diff
@@ -27,102 +28,65 @@ def add_faz(self):
             self.fazan = self.fazan +'n'
         self.scor += 1
 
-
 class Calculator(Player):
      def __init__(self):
          super(Calculator,self).__init__("Calculator")
      def my_turn(self,pref):
-        f = open(pref + '.txt','r+')
-        s = 0
-        lines = []
-        for line in f:
-            lines.append(line)
-        f.seek(0)
-        rand = random.randrange(len(lines))
-        cuv = lines[rand]
-        del lines[rand]
-        for line in lines:
-            f.write(line)
-        f.close()
+        global word_list
+        cuv = random.choice(word_list[pref])
+        word_list[pref].remove(cuv)
         return cuv
 
+files_in_dir = os.listdir('.')   #a list containing the names of the entries in the current directory
```

Comment:
Use proper commenting style: make it a sentence, start with Capitalized word, end with dot. Leave space between # and comment.